releng/source-tarball/build.bash

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+function printHelp() {
+  >&2 echo \
+"USAGE: $0 [-p INFLUXDB_GIT_DIR] [-S GITHUB_SSH_KEY]
```

Comment:
We don't need the `-S` flag in this script since there are no private repos to clone for influxdb.